homeassistant/components/binary_sensor/vera.py

```diff
@@ -0,0 +1,95 @@
+"""
+Support for Vera binary sensors.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/binary_sensor.vera/
+"""
+import logging
+
+import homeassistant.util.dt as dt_util
+from homeassistant.const import (
+    ATTR_ARMED, ATTR_BATTERY_LEVEL, ATTR_LAST_TRIP_TIME, ATTR_TRIPPED)
+from homeassistant.components.binary_sensor import (
+    BinarySensorDevice)
+from homeassistant.loader import get_component
+
+REQUIREMENTS = ['pyvera==0.2.8']
```

Comment:
Shouldn't this be a dependency on the vera component instead of a requirement for the package? Same for all other platforms